src/bluesky_stomp/messaging.py

```diff
@@ -384,10 +401,18 @@ def _on_message(self, frame: Frame) -> None:
             Mapping[str, Any],
             frame.headers,  # type: ignore
         )
+
+        trace_context = retrieve_context_from_stomp_headers(frame)
+
+        tracer = get_tracer("_on_message")
+
         if (sub_id := headers.get("subscription")) is not None:
             if (sub := self._subscriptions.get(sub_id)) is not None:
                 try:
-                    sub.callback(frame)
+                    with tracer.start_as_current_span(
+                        sub.callback.__name__, trace_context
+                    ):
+                        sub.callback(frame)
```

Comment:
Need to add tracing to subscribe, capturing the destination and callback function name, as well as connect and disconnect if possible adding an attrubute to indicate sucess when the connect/disconnect is complete?